apiserver/facades/client/charmhub/convert.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package charmhub
+
+import (
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/charmhub"
+)
+
+func convertCharmInfoResult(info charmhub.InfoResponse) params.InfoResponse {
```

Comment:
How about `transform`?

Reply:
I do like transform.  Juju seems to be inconsistent with naming these types of functions, between convertBlah and transforBlah.  ConvertBlah is significantly more prevelent.